Proctor_Brad/ruby_oop/animals/mammal.rb

```diff
@@ -0,0 +1,13 @@
+class Mammal
+    attr_accessor :alive
+    def initialize
+        @alive = true
+        puts 'I am alive!'
+        self

```

Comment:
Initialize doesn't need to return self, it's technically called by new :) so it's return wouldn't matter.